guides/common/modules/proc_registering-a-host.adoc

```diff
@@ -49,7 +49,15 @@ endif::[]
 .Procedure
 . In the {ProjectWebUI}, navigate to *Hosts* > *Register Host*.
 . Enter the details for how you want the registered host to be configured.
+* If you select a host group from the *Host Group* list, the following fields inherit their values from the host group:
++
+* *Operating system*
+ifdef::katello,satellite,orcharhino[]
+* *Activation Keys*
+endif::[]
+* *Lifecycle environment*
 ifdef::katello,satellite,orcharhino[]
+* A {SmartProxy} behind a load balancer takes precedence over the {SmartProxy} selected in the {ProjectWebUI} as the content source of the host.
```

Comment:
Eh, now the indentation is wrong:
![Screenshot From 2025-01-17 12-17-44](https://github.com/user-attachments/assets/fb677297-59b0-428d-9527-ce3a3cdf9021)
This is a problem, not just a nitpick IMO. @rh-max please try this or anything else you find appropriate:
```suggestion
* If you select a host group from the *Host Group* list, the following fields inherit their values from the host group:
+
** *Operating system*
ifdef::katello,satellite,orcharhino[]
** *Activation Keys*
endif::[]
** *Lifecycle environment*
ifdef::katello,satellite,orcharhino[]
* A {SmartProxy} behind a load balancer takes precedence over the {SmartProxy} selected in the {ProjectWebUI} as the content source of the host.
```

Reply:
@asteflova Thanks for the catch(es), the list LGTM now!